junit5-jupiter-starter-bazel/WORKSPACE

```diff
@@ -1,13 +1,48 @@
-load(":junit5.bzl", "junit_jupiter_java_repositories", "junit_platform_java_repositories")
-
 JUNIT_JUPITER_VERSION = "5.9.1"
-
 JUNIT_PLATFORM_VERSION = "1.9.1"
 
-junit_jupiter_java_repositories(
-    version = JUNIT_JUPITER_VERSION,
+RULES_JVM_EXTERNAL_VERSION = "4.2"
+RULES_JVM_EXTERNAL_SHA = "cd1a77b7b02e8e008439ca76fd34f5b07aecb8c752961f9640dea15e9e5ba1ca"
```

Comment:
There is a newer version of rules_jvm_external: https://github.com/bazelbuild/rules_jvm_external/releases/tag/4.4.2

Reply:
Thanks for pointing that out! Updated in 740487b35a38d1c79a870b46bb67afbc8ad7c6b8.